### ISSUE TITLE
Chyba po aktualizaci

### ISSUE THREAD

**Issue by modern74** [amaximus/garbage-collection-card#47]
Dobrý den,

po aktualizaci na nejnovější verzi 1.10.1 se vyskytla tato chyba.
![Garbage](https://user-images.githubusercontent.com/61018989/94838858-7abe1000-0416-11eb-9fe7-d1e116d3f592.jpeg)

Vrátil jsem se zpět k předešlé verzi, tam je to v pořádku.

Děkuji
Petr

**Comment by amaximus** [comment]
Are you using the card within another card? (e.g. stacked vertically or horizontally?)

The setConfig function is invoked by HASS frontend, it's not actively used in the card itself.

Do you have the Lovelace resource defined in the configuration.yaml?

```
lovelace:
  resources:
    - {type: js, url: '/hacsfiles/garbage-collection-card/garbage-collection-card.js'}
```

**Comment by amaximus** [comment]
The above solution was confirmed to fix issue. Could you please try it out and get back to me with results?

**Comment by modern74** [comment]
Same error even after upgrading to version 1.12.0
I have HA version 0.116.2

**Comment by amaximus** [comment]
Yes, but do you have the following lines in your configuration.yaml? If not, please add them.

```
lovelace:
  resources:
    - {type: js, url: '/hacsfiles/garbage-collection-card/garbage-collection-card.js'}
```

**Comment by modern74** [comment]
of course, I have added in configuration.yaml

but I have also added in HA sources .... 
![stažený soubor](https://user-images.githubusercontent.com/61018989/95766356-7239c500-0cb3-11eb-909d-bb13a0fb217a.jpeg)


When I tried to delete this, the app did not load at all.

**Comment by amaximus** [comment]
Could you please clear your browser can and shift-reload the page? (or ctrl-reload depending on the browser)

**Comment by modern74** [comment]
yes, but no change.
the latest functional version is 1.10.0.
all other versions don't work for me.

**Comment by amaximus** [comment]
After adding the lovelace resource did you restart Home Assistant? Updating configuration.yaml usually needs a restart.

**Comment by modern74** [comment]
Yes, everytime

**Comment by modern74** [comment]
It doesn't work in the new version 1.15.0 
![garbage](https://user-images.githubusercontent.com/61018989/97589129-79740900-19fd-11eb-97fa-c91a0454c2a9.jpeg)
 either.
I enclose an error extract from the log.

**Comment by amaximus** [comment]
Thanks for the error message, this helped a lot.
Line 16 [1] indicated issue with the language file that is in JSON format. Presuming that you are using the Czech translation, I checked the cs.json and indeed the JSON was missing a closing curl bracket as per [2].

I fixed it in the new 1.15.1 release.
Please update your card and get back to me with results.
 
[1] https://github.com/amaximus/garbage-collection-card/blob/master/dist/garbage-collection-card.js#L16
[2] https://github.com/amaximus/garbage-collection-card/commit/7dbf8176662666551d095c4cd3c442cce7d18c92

**Comment by modern74** [comment]
Thank you for a quick response :)
I have to confirm that in this new version 1.15.1 the card is already working properly.
Many thanks!
